src/libraries/System.Diagnostics.Debug/tests/System.Diagnostics.Debug.Tests.csproj

```diff
@@ -26,4 +18,13 @@
     <Compile Include="EmptyAttributeTests.cs" />
     <Compile Include="XunitAssemblyAttributes.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="$(CoreLibProject)" />
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Runtime\src\System.Runtime.csproj;
+                               $(LibrariesProjectRoot)System.Threading\src\System.Threading.csproj"
+                      SkipUseReferenceAssembly="true" />
```

Comment:
We should document this `SkipUseReferenceAssembly` flag and when it should be used.  Unrelated to this PR.

I wonder if this is an opportunity for standardization: https://github.com/dotnet/msbuild/blob/aed5e7ed0b7e031d3e486c63b206902bf825b128/src/Tasks/Microsoft.Common.CurrentVersion.targets#L2197-L2214